src/main/kotlin/com/example/fhirvalidator/util/OperationOutcome.kt

```diff
@@ -0,0 +1,25 @@
+package com.example.fhirvalidator.util
+
+import org.hl7.fhir.r4.model.OperationOutcome
+import java.util.*
```

Comment:
not your code but is the whole util package needed? or is `java.util.Collections` sufficient